config/stack_noaa.yaml

```diff
@@ -25,12 +25,12 @@ szip:
 jasper:
   build: YES
   shared: NO
-  version: 2.0.25
+  version: 2.0.22
 
 udunits:
-  build: YES
+  build: NO
```

Comment:
please revert to develop

Reply:
I can turn it back on, however please see the correspondence below between myself and @kgerheiser form an email back in January/February:

Julie: 
> I successfully ran:
> ./setup_modules.sh -p /gpfs/dell2/emc/verification/noscrub/Julie.Prestopnik/hpc-stack/git/test -c config/config_wcoss_dell_p3.sh
> 
> but when I tried to run:
> ./build_stack.sh -p /gpfs/dell2/emc/verification/noscrub/Julie.Prestopnik/hpc-stack/git/test -c config/config_wcoss_dell_p3.sh -y config/stack_custom.yaml -m
> 
> I received the following error:
> + cmake .. -DCMAKE_INSTALL_PREFIX=/gpfs/dell2/emc/verification/noscrub/Julie.Prestopnik/hpc-stack/git/test/ips-18.0.1.163/udunits/2.2.26 -DBUILD_SHARED_LIBS=OFF
> CMake Error at /gpfs/dell1/usrx/local/prod/packages/cmake/3.16.2/share/cmake-3.16/Modules/CMakeDetermineCCompiler.cmake:49 (message):
>   Could not find compiler set in environment variable CC:
>   icc.
> Call Stack (most recent call first):
>   CMakeLists.txt:3 (PROJECT)
> 
> CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
> 
> I don't think I need udunits built for MET, but I don't think it's appropriate to change config/stack_custom.yaml to not build UDUNITS, so I imagine I need to set "CMAKE_C_COMPILER".  I'm not sure where to set that, however.  I don't see that set in any other files:
> [Julie.Prestopnik@m71a1 libs]$ grep -i CMAKE_C_COMPILER *
> [Julie.Prestopnik@m71a1 libs]$ cd ../config
> [Julie.Prestopnik@m71a1 config]$ grep -i CMAKE_C_COMPILER *
> [Julie.Prestopnik@m71a1 config]$ 
> 
> What am I misunderstanding?

Kyle:
> You're right about Udunits not building. I can reproduce it and I'm not sure what the problem is. I'm looking into it.

Kyle:
> I read more closely about your Udunits issue. I would just turn it off (build: NO). I'll investigate why it's not working there (when it works other places), but it's not necessary for this work.

Do you still want me to turn this on (build: YES)?